lib/api/fields/types/image.php

```diff
@@ -82,3 +82,37 @@ function _beans_get_image_id_attributes( $id, array $field, $is_multiple ) {
 
 	return $attributes;
 }
+
+/**
+ * Get the image's URL and alt values.
+ *
+ * @since 1.5.0
+ *
+ * @param mixed $image_id The image's attachment ID.
+ *
+ * @return array|void
+ */
+function _beans_get_image_url_and_alt( $image_id ) {
```

Comment:
Why does this function have the logic to do two things? Split it into two functions, and if necessary, have this function call those two functions and return the array.

It's currently being used in `list()` anyway, so the variables could have individual assignments.